launchable/test_runners/raw.py

```diff
@@ -150,8 +150,12 @@ def parse_json(test_result_file: str) -> Generator[CaseEventType, None, None]:
                 test_path, duration_secs, CaseEvent.STATUS_MAP[status],
                 case['stdout'], case['stderr'], created_at)
 
-    if not test_result_file.endswith('.xml'):
-        client.parse_func = parse_json
+    for test_result_file in test_result_files:
+        if not test_result_file.endswith('.xml'):
+            client.parse_func = parse_json
```

Comment:
[question]
Don't we have to determine if the file ends with `.json`?
Or is it filtered beforehand?

Reply:
I found out. It's a historical reason.
We support only JSON format at first. After that, we started to support XML file format.
So, if the report file isn't XML, we set a json format parser.

https://github.com/launchableinc/cli/pull/505

Reply:
We only support JSON and XML, so we don't need to check an extension is `.json` when it isn't `.xml`.